tools/flatpak/io.github.randovania.Randovania.yml

```diff
@@ -0,0 +1,46 @@
+app-id: io.github.randovania.Randovania
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [ org.freedesktop.Sdk.Extension.mono6 ]
+finish-args:
+    - --device=dri          # UI will want hw acceleration
+    - --share=network       # Multiworld support
+    - --share=ipc           # Needed for X11 support
+    - --socket=wayland
+    - --socket=fallback-x11
```

Comment:
Tested the latest artifact on Steam Deck and can confirm it works, with one added line:

```suggestion
    - --socket=fallback-x11
    - --filesystem=xdg-run/app/com.discordapp.Discord:create
```

This makes it so that the login system can detect Discord even when both are in Flatpak containers; we should probably document this somewhere in the README (maybe the UI too, when `/.flatpak` is detected?):

https://github.com/flathub/com.discordapp.Discord/wiki/Rich-Precense-(discord-rpc)

After this was done, was able to log in and generate possibly the first seed made on Deck hardware!

![image](https://user-images.githubusercontent.com/1396814/213903783-972f32e4-3885-44f9-a121-5203fe46dce2.png)